app/src/main/java/com/vuongnq14798/vuongmp3/ui/mymusic/MyMusicFragment.kt

```diff
@@ -1,15 +1,63 @@
 package com.vuongnq14798.vuongmp3.ui.mymusic
 
 import android.os.Bundle
+import android.widget.Toast
+import com.vuongnq14798.vuongmp3.MainActivity
 import com.vuongnq14798.vuongmp3.R
 import com.vuongnq14798.vuongmp3.base.BaseFragment
+import com.vuongnq14798.vuongmp3.data.model.Track
+import com.vuongnq14798.vuongmp3.data.source.TracksRepository
+import com.vuongnq14798.vuongmp3.data.source.local.TracksLocalDataSource
+import com.vuongnq14798.vuongmp3.data.source.remote.TracksRemoteDataSource
+import com.vuongnq14798.vuongmp3.ui.playmusic.PlayMusicActivity
+import com.vuongnq14798.vuongmp3.util.TrackDiffCallBack
+import kotlinx.android.synthetic.main.fragment_my_music.*
+import kotlinx.android.synthetic.main.fragment_my_music.fab
+
+class MyMusicFragment : BaseFragment(),
+    MyMusicContract.View,
+    TrackAdapter.OnTrackClickListener {
+
+    private val tracksRepository: TracksRepository by lazy {
+        TracksRepository(
+            TracksRemoteDataSource.getInstance(),
+            TracksLocalDataSource.getInstance(context!!)
+        )
```

Comment:
context?.let{
    TracksRepository(
            TracksRemoteDataSource.getInstance(),
            TracksLocalDataSource.getInstance(it)
        )
}